operations/host_spawn.go

```diff
@@ -1516,7 +1548,7 @@ func hostFindBy() cli.Command {
 			hostUser := utility.FromStringPtr(host.StartedBy)
 			link := fmt.Sprintf("%s/host/%s", conf.UIServerHost, hostId)
 			fmt.Printf(`
-	     ID : %s
+		 ID : %s
```

Comment:
was this intentional?

Reply:
No, vim just auto-reindents files when files have mixed tabs/spaces used in indentation and this output string has a mix of tabs/spaces. I reverted this change.